blivet/formats/swap.py

```diff
@@ -21,8 +21,10 @@
 #
 
 from parted import PARTITION_SWAP, fileSystemType
+from ..errors import FSWriteUUIDError
```

Comment:
I know, the commit message says "other file systems" and I should have just used "file systems".